atlas/additional-guides/build-notifications.mdx

```diff
@@ -0,0 +1,18 @@
+There's a number of ways you can get notified about finished Atlas builds
+
+## E-mail notifications based on GitHub comments
+Atlas posts comments on commit pages once a build is complete. These comments contain information about status of the build as well as link to the build logs and to the Atlas environment.
+
+![GitHub comment posted by Atlas](./assets/img/notifications/gh-comment.png)
+
+If you wish to receive an e-mail update once this happens, make sure you configure your [GitHub settings](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#customizing-your-email-notifications) accordingly.
+
+![GitHub settings panel](./assets/img/notifications/gh-settings.png)
+
+
+## Notifications in the User Portal
+Once your build is finished, you'll be notified about this event in the Atlas User Portal. To browse notifications, click on the bell icon on top of the page:
+
+![Atlas notifications in Portal](./assets/img/notifications/portal-notifications.png)
+
+Clicking on `View build...` button is going to take you to the build logs page.
```

Comment:
Adding "the" :

_Clicking on the `View build...` button is going to take you to the build logs page._